packages/router/src/config.ts

```diff
@@ -1,4 +1,5 @@
 ///NXTP Config Generator based on vector/modules/router/src/config.ts
+import https from "https";
```

Comment:
Ethers already has a `fetchJson` method that is better for this use case.

Reply:
I almost used that method but I was concerned bc the method seemed to be doing unnecessary work for our case